cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -548,27 +540,24 @@ def test_guest_os_reset(appliance, testing_vm_tools, ensure_vm_running, soft_ass
     testing_vm_tools.power_control_from_cfme(
         option=testing_vm_tools.GUEST_RESTART, cancel=False, from_details=True)
     view.flash.assert_success_message(text='Restart Guest initiated', partial=True)
-
-    soft_assert(
-        wait_for_last_boot_timestamp_refresh(testing_vm_tools, last_boot_time),
-        "Last Boot Time value has not been refreshed")
+    if not (BZ(1571830, forced_streams=["5.10", "5.11"]).blocks and provider.one_of(RHEVMProvider)):
```

Comment:
If you check provider type first, it saves the time it takes to then check `BZ.blocks`